src/modules/addMALscore.js

```diff
@@ -114,20 +114,14 @@ async function addMALscore(type,id){
 					adder()
 				}
 			}
-			if(window.GM_xmlhttpRequest){
+			if(GM_xmlhttpRequest){
 				GM_xmlhttpRequest({
 					method: "GET",
 					anonymous: true,
 					url: "https://myanimelist.net/" + type + "/" + data.data.Media.idMal + "/placeholder/userrecs",
 					onload: function(response){handler(response)}
 				})
 			}
-			else{
-				let oReq = new XMLHttpRequest();
-				oReq.addEventListener("load",function(){handler(this)});
-				oReq.open("GET","https://myanimelist.net/" + type + "/" + data.data.Media.idMal + "/placeholder/userrecs");
-				oReq.send()
-			}
```

Comment:
This still works AFAIK in some browsers.

Reply:
Huh, I didn't know that

Reply:
added it back, though it wasn't working for me when forcing xhr in FF or Edge